File: Naloga 17/naloga_17.py
def naloga_17():
    def deljivost(x, y):
        if x % y == 0:
            return True
    for stevilo in range(1, 300):
        if deljivost(stevilo, 2):
            if deljivost(stevilo, 7):
                print(stevilo)

if __name__ == '__main__':
    naloga_17()

File: Naloga 17/naloga_17.cpp
#include <iostream>

bool deljivost(int x, int y) {
    if ((x % y) == 0) {
        return true;
    }
    return false;
}
void naloga_17() {
    for (int stevilo = 1; stevilo<=300; ++stevilo) {
        if (deljivost(stevilo, 2)) {
            if (deljivost(stevilo, 7)) {
                std::cout << stevilo << std::endl;
            };
        };
    };
};

int main() {
    naloga_17();
    return 0;
};